Functional.Tests/Options/OptionExtensionsTests.cs

```diff
@@ -12,8 +12,8 @@ public class OptionExtensionsTests
 	{
 		public class WhenOptionIsSome
 		{
-			private static Option<int> Value => Option.Some(10);
-			private static Option<int> NoValue => Option.None<int>();
+			private const int IntValue = 10;
```

Comment:
I like that you dried it and re-use the same variable.
I'd prefer Value and WrappedValue rather than hungarian notation. Or _value and Value would work too. Any specific reason you prefer CamelCase for private consts?

Reply:
I prefer `ALL_CAPS` for private constants.  This is just what ReSharper told me to do lol.